<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplicationSettingsWidgetGeneral</class>
 <widget class="QWidget" name="ApplicationSettingsWidgetGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>881</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="generalSettingsTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>Basic Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="startupGroup">
         <property name="title">
          <string>Startup</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="singleInstanceCheckBox">
            <property name="text">
             <string>Start only a single instance of KeePassXC</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="rememberLastDatabasesCheckBox">
            <property name="text">
             <string>Remember last databases</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="rememberLastKeyFilesCheckBox">
            <property name="text">
             <string>Remember last key files and security dongles</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="openPreviousDatabasesOnStartupCheckBox">
            <property name="text">
             <string>Load previous databases on startup</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="systrayMinimizeOnStartup">
            <property name="text">
             <string>Minimize window at application startup</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkForUpdatesOnStartupCheckBox">
            <property name="text">
             <string>Check for updates at application startup</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="saveGroup">
         <property name="title">
          <string>File Management</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="useAtomicSavesCheckBox">
            <property name="text">
             <string>Safely save database files (may be incompatible with Dropbox, etc)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="backupBeforeSaveCheckBox">
            <property name="text">
             <string>Backup database file before saving</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="autoSaveAfterEveryChangeCheckBox">
            <property name="text">
             <string>Automatically save after every change</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="autoSaveOnExitCheckBox">
            <property name="text">
             <string>Automatically save on exit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ignoreGroupExpansionCheckBox">
            <property name="text">
             <string>Don't mark database as modified for non-data changes (e.g., expanding groups)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="autoReloadOnChangeCheckBox">
            <property name="text">
             <string>Automatically reload the database when modified externally</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="entryGroup">
         <property name="title">
          <string>Entry Management</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QCheckBox" name="useGroupIconOnEntryCreationCheckBox">
            <property name="text">
             <string>Use group icon on entry creation</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="minimizeOnCopyCheckBox">
            <property name="text">
             <string>Minimize when copying to clipboard</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="previewHideCheckBox">
            <property name="text">
             <string>Hide the entry preview panel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="generalGroup">
         <property name="title">
          <string>General</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QCheckBox" name="toolbarHideCheckBox">
            <property name="text">
             <string>Hide toolbar (icons)</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="toolbarMovableLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <item>
             <spacer name="toolbarMovableSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="toolbarMovableCheckBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Movable toolbar</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="toolButtonStyleLayout">
            <property name="spacing">
             <number>15</number>
            </property>
            <item>
             <spacer name="toolButtonStyleSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item alignment="Qt::AlignRight">
             <widget class="QLabel" name="toolButtonStyleLabel">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Button style</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="toolButtonStyleComboBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="minimizeOnCloseCheckBox">
            <property name="text">
             <string>Minimize instead of app exit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="systrayShowCheckBox">
            <property name="text">
             <string>Show a system tray icon</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="systrayDarkIconCheckBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Dark system tray icon</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="systraySettings" native="true">
            <layout class="QVBoxLayout" name="systrayLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="sizeConstraint">
                <enum>QLayout::SetMaximumSize</enum>
               </property>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QCheckBox" name="systrayMinimizeToTrayCheckBox">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Hide window to system tray when minimized</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="languageLabelLayout_2">
            <property name="spacing">
             <number>15</number>
            </property>
            <item alignment="Qt::AlignRight">
             <widget class="QLabel" name="languageLabel_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Language</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="languageComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAutotype">
      <attribute name="title">
       <string>Auto-Type</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="autoTypeEntryTitleMatchCheckBox">
         <property name="text">
          <string>Use entry title to match windows for global Auto-Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoTypeEntryURLMatchCheckBox">
         <property name="text">
          <string>Use entry URL to match windows for global Auto-Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoTypeAskCheckBox">
         <property name="text">
          <string>Always ask before performing Auto-Type</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="topMargin">
          <number>10</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="autoTypeShortcutLabel">
           <property name="text">
            <string>Global Auto-Type shortcut</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="ShortcutWidget" name="autoTypeShortcutWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="autoTypeDelayLabel">
           <property name="text">
            <string>Auto-Type typing delay</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="autoTypeDelaySpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="suffix">
            <string comment="Milliseconds"> ms</string>
           </property>
           <property name="prefix">
            <string/>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>25</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="autoTypeStartDelayLabel">
           <property name="text">
            <string>Auto-Type start delay</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="autoTypeStartDelaySpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="suffix">
            <string comment="Milliseconds"> ms</string>
           </property>
           <property name="prefix">
            <string/>
           </property>
           <property name="minimum">
            <number>100</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>500</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShortcutWidget</class>
   <extends>QLineEdit</extends>
   <header>autotype/ShortcutWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
